cmd/observer/node_builder/observer_builder.go

```diff
@@ -571,6 +591,10 @@ func (builder *ObserverServiceBuilder) extraFlags() {
 			defaultConfig.upstreamNodePublicKeys,
 			"the networking public key of the upstream access node (in the same order as the upstream node addresses) e.g. \"d57a5e9c5.....\",\"44ded42d....\"")
 		flags.BoolVar(&builder.rpcMetricsEnabled, "rpc-metrics-enabled", defaultConfig.rpcMetricsEnabled, "whether to enable the rpc metrics")
+		flags.BoolVar(&builder.executionDataIndexingEnabled,
+			"execution-data-indexing-enabled",
+			defaultConfig.executionDataIndexingEnabled,
+			"whether to enable the execution data indexing")
```

Comment:
I think we'll need all of these flags:
https://github.com/onflow/flow-go/blob/ae8ab050acd8d73f2fdf475aad053eb00ece49e7/cmd/access/node_builder/access_node_builder.go#L1119-L1125

Reply:
Thanks for noticing the missing flags.